packages/react-components/babel-preset-global-context/jest.config.js

```diff
@@ -1,4 +1,23 @@
 // @ts-check
+/* eslint-disable */
+
+const { readFileSync } = require('node:fs');
```

Comment:
I'm wondering. Can this repeated code be extracted to a utils function and be called here (and in the other packages too)?

Reply:
it probably can, but I try to avoid it:
- it's generated boilerplate (maintained by automation) that once/IFF swc will provide better resolution gymnastics will be removed.
- I dont wanna introduce a dependency on workspace package for simple jest setup

in future we might somehow aggregate repetition for both swc and swc test